### PR TITLE
New package: hosts-update

### PR DESCRIPTION
**Pull request by renatoaguiar**

### CODE REVIEW

File: srcpkgs/hosts-update/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'hosts-update'
+pkgname="hosts-update"
+version="1.37"
+revision=1
+build_style=gnu-makefile
+short_desc="Updates /etc/hosts with the mvps blocklist"
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="MIT"
+homepage="https://github.com/graysky2/hosts-update"
+distfiles="https://github.com/graysky2/hosts-update/archive/v1.37.tar.gz"
```
Comment (xtraeme) [inline srcpkgs/hosts-update/template]: `1.37` -> `${version}`

File: srcpkgs/hosts-update/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+checksum="aee3e537b85f9300761a543345f3a6331b07c8f2cd42511a01e9c008c6379859"
+depends="bash curl"
+configure_args="--with-udevrulesdir=/usr/lib/udev/rules.d/"
```
Comment (xtraeme) [inline srcpkgs/hosts-update/template]: `configure_args` is ignored with `build_style=gnu-makefile`

**Comment by xtraeme** [comment]
Missing `noarch=yes`

File: srcpkgs/hosts-update/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'hosts-update'
+pkgname="hosts-update"
+version="1.37"
```
Comment (Vaelatern) [inline srcpkgs/hosts-update/template]: Quotes not needed for either of these variables.

**Comment by Vaelatern** [comment]
79c7427

**Pull request closed without merging**